irohad/ametsuchi/impl/peer_query_wsv.cpp

```diff
@@ -15,21 +15,38 @@
  * limitations under the License.
  */
 
+#include "builders/protobuf/common_objects/proto_peer_builder.hpp"
 #include "ametsuchi/impl/peer_query_wsv.hpp"
 
-#include <utility>
-
 #include "ametsuchi/wsv_query.hpp"
-#include "model/peer.hpp"
+
+using wPeer = std::shared_ptr<shared_model::interface::Peer>;
```

Comment:
`w` prefix looks not suitable here.

Reply:
Also, what is the point for making this `using` if same using present into `.hpp` file?